### PR TITLE
add linear & example

### PR DESCRIPTION
**Pull request by justyx404**
Want to use this short one as an example to see if its what you expected and how you would like the examples to be organized.

### CODE REVIEW

**Comment by wozeparrot** [comment]
Yea pretty much, I'm using `#->` after the prints, kinda like an anchor, with the expected output after that so that tooling can eventually test it.

**Comment by justyx404** [comment]
That sounds good. Also reduced the size of Tensor so the printed output would be manageable.

File: docs/api/tinygrad/tensor.md [hunk 1]
```diff
@@ -439,3 +439,28 @@ print(t.grad.numpy()) #->
...
 """
 ```
+#### `.linear(weight:Tensor, bias:Optional[Tensor]=None)`
+
+Applies a linear transformation to the current tensor.
+
+```python
+t = Tensor.randn(5,2)
+w = Tensor.randn(2,3)
+b = Tensor.randn(5,3)
```
Comment (wozeparrot) [inline docs/api/tinygrad/tensor.md]: probably move this to before where its actually used

File: docs/api/tinygrad/tensor.md [hunk 2]
```diff
@@ -480,5 +480,29 @@ print(t2.numpy()) #->
...
+print(t.linear(w, b).numpy()) #->
+"""
+[[ 3.360947   -0.05869228 -1.0911431 ]
+ [ 1.6087391   2.7058008   2.1051977 ]
+ [-0.19086033 -0.61030436 -1.898639  ]
+ [-3.275361    0.4857852   1.0790952 ]
+ [ 1.9292889  -1.3569635  -0.12129784]]
+ """
+ ```
```
Comment (wozeparrot) [inline docs/api/tinygrad/tensor.md]: accidental indent?

Comment (justyx404) [reply docs/api/tinygrad/tensor.md]: ops, my bad.

**Pull request merged**